invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/side_bar/external_resources.html

```diff
@@ -0,0 +1,51 @@
+{#
+  Copyright (C) 2020 CERN.
+  Copyright (C) 2020 Northwestern University.
+  Copyright (C) 2021 TU Wien.
+  Copyright (C) 2022 New York University.
+
+  Invenio RDM Records is free software; you can redistribute it and/or modify
+  it under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+{%- if config.get("APP_RDM_RECORD_LANDING_PAGE_EXTERNAL_LINKS") and external_resources | length > 0 -%}
+{# if no external resources exist, don't bother showing the box #}
+<section id="external-resource" aria-label="{{ _('External resources') }}" class="ui segment rdm-sidebar external resource">
+  <h2 class="ui small header">{{ _('External resources')}}</h2>
+  {# dynamically create the list of external resources #}
+
+  <ul class="details-list m-0 p-0" style="list-style-type: none;">
+    {%- for resource in external_resources -%}
+      <li id="license-{{resource.id}}-{{loop.index}}" class="has-popup">
+        <div id="title-{{resource.id}}-{{loop.index}}"
+             class="license clickable"
+             tabindex="0"
+             aria-haspopup="dialog"
+             aria-expanded="false"
+             role="button"
+             aria-label="{{ resource.title_l10n }}"
+             style="display:flex; flex-direction: row;">
+
+          {% if resource.icon %}
+            {% set iconFile = 'icons/external_resources/{}.svg'.format(resource.icon) %}
+            <span class="icon-wrap">
+              <img class="icon" src="{{ url_for('static', filename=iconFile) }}" alt="{{ resource.id }} icon"/>
+            </span>
+          {% endif %}
+            <div style="display: flex; flex-direction: column; justify-content: space-around;">
+              <span class="title-text">
+                <a href="{{resource.link}}" target="_blank">{{ resource.title_l10n }}</a>
```

Comment:
When using `target="_blank"`, I would also add [noreferrer](https://stackoverflow.com/questions/50773152/when-should-i-use-rel-noreferrer) (check if there are other occurrences).